README.md

```diff
@@ -390,6 +390,16 @@ Machine.successors
 }
 ```
 
+## Class constants
+Each machine's state will turn into a constant:
+```ruby
```

Comment:
Would be good to give an example of a simple state machine here, e.g.

```ruby
class OrderStateMachine
  include Statesman::Machine

  state :pending, initial: true
  state :checking_out
  state :cancelled

  # Constants created as a side effect of adding state
  transition from: PENDING, to: [CHECKING_OUT, CANCELLED]
end
```
Which shows how why the constants are more useful - i.e. error at app boot rather than runtime.